apps/strapi-cms/src/components/banner-links/banner-links.json

```diff
@@ -0,0 +1,22 @@
+{
+  "collectionName": "components_banner_links_banner_links",
+  "info": {
+    "displayName": "bannerLinks",
+    "icon": "file"
+  },
+  "options": {},
+  "attributes": {
+    "name": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "body": {
+      "type": "blocks"
+    },
+    "theme": {
+      "type": "json"
+    }
```

Comment:
I think this property is dangerous and undefined (what kind of json do you expect? what if the json is not properly formatted?), we should replace it with a more simple prop, based on the specific need

Reply:
I thought we could use that to have a style object, but it's safer to remove that property and to add a specific filed when we need it